docs/user-guide/release-notes/2.11.x.rst

```diff
@@ -13,4 +13,7 @@ New Features
   since the last sync.
 * The yum_importer can sync a distribution unit even when no yum metadata is present in the repo.
   This is useful for kickstart trees related to Project Atomic.
-* Added support for publishing ISO repositories to custom relative paths.
\ No newline at end of file
+* Added support for publishing ISO repositories to custom relative paths.
+* RPMs are now being published into new folder structure e.g. "Packages/d/dnf.rpm", this applies
+  to all RPMs which will be published after this update even during incremental publish in repo
+  with old folder structure.
```

Comment:
Add "the" before "old".

I also would add something similar to the following sentence to be very clear about the behavior: "Incremental publishes will place newly-added RPMs into the new folder structure while leaving previously-published RPMs in the old locations. This is safe and correct. If you prefer to have all RPMs published in the new layout, simply perform a full publish on each repository."